testing.md

```diff
@@ -0,0 +1,79 @@
+# Testing
+
+## Best Practices
+
+Writing good tests in javascript is similar to writing good tests in any other language. There are a few best practices that are more pertinent to javascript testing.
+
+### Mocking
+
+Mock any dependency with `sinon.js`. `sinon.js` is extremely easy to use.
+TODO
+
+### No AJAX calls
+TODO
```

Comment:
this one seems to stand on it's own :)  (unless you want to justify why ajax in tests is a bad idea)